popmon/visualization/overview_section.py

```diff
@@ -37,14 +39,17 @@ class OverviewSectionGenerator(Module):
     which later will be used for the report generation.
```

Comment:
The overview section description is now not accurate anymore ("Alerts aggregated per feature").
An example of what we could put is: "Overview of the dataset, analysis and alerts."